README.md

```diff
@@ -1,7 +1,19 @@
-This overlay contains CoreOS specific packages and Gentoo packages
+This overlay contains Container Linux specific packages and Gentoo packages
 that differ from their upstream Gentoo versions.
 
+See the [portage-stable](https://github.com/coreos/portage-stable) repo
+for packages which do not have Container Linux specific changes.
+
 Licensing information can be found in the respective files, so consult
 them directly. Most ebuilds are licensed under the GPL version 2.
 
 Upstream Gentoo sources: https://gitweb.gentoo.org/repo/gentoo.git
+
+`coreos-base/coreos` is the package responsible for everything that gets
+built into an image.
+
+`coreos-devel/board-packages` is everything that could be built into a
+development or production image, plus any OEM specific packages.
```

Comment:
There are a bunch of OEM packages not listed in `board-packages`.  Should we add them?

Reply:
Uh probably? I just found this but that seems to fit. Maybe @dm0- knows better?